src/java.desktop/share/classes/sun/print/PrintJob2D.java

```diff
@@ -28,47 +28,14 @@
 import java.awt.Dimension;
 import java.awt.Frame;
 import java.awt.Graphics;
-import java.awt.Graphics2D;
 import java.awt.PrintJob;
 import java.awt.JobAttributes;
 import java.awt.JobAttributes.*;
 import java.awt.PageAttributes;
 import java.awt.PageAttributes.*;
```

Comment:
wildcard

Reply:
yes, they aren't used in this class. I'll remove them.